src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp

```diff
@@ -500,15 +507,52 @@ int XMC_Flasher::writeReg(unsigned RegOffset, unsigned value) {
     return 0;
 }
 
+static std::string getStatus(int status, std::map<int, std::string> &map)
+{
+    auto entry = map.find(status);
+    std::ostringstream os;
+
+    os << std::hex << entry->first;
+
+    if (entry != map.end())
+	    os << "(" << entry->second << ")";
+
+    return os.str();
+}
+
+/* XMC means CMC */
+void XMC_Flasher::initCMCStatusMap()
+{
+    cmcStatusMap = {
+        {0, "NOT READY"},
+        {1, "READY"},
+        {2, "STOPPED"},
+        {4, "PAUSED"},
+    };
+}
+
+/* BMC means SC */
+void XMC_Flasher::initSCStatusMap()
+{
+    scStatusMap = {
+        {0, "NOT READY"},
+        {1, "READY"},
+        {2, "BSL_UNSYNCED"},
+        {3, "BSL_SYNCED"},
+        {4, "BSL_SYNCED_SC_NOT_UPGRADABLE"},
+        {5, "READY_SC_NOT_UPGRADABLE"},
+    };
+}
+
```

Comment:
We don't need a function to init the maps. We can simply define static std::map<int, std::string> maps

Reply:
fixed